prisma/schema.prisma

```diff
@@ -34,11 +34,17 @@ model Experiences {
 
 model Places {
   id           Int      @id @default(autoincrement())
-  place_type   PlaceType     @default(ONSEN) // NOTE Needs to be more strict
+  place_type   PlaceType     @default(ONSEN)
   name         String   @db.VarChar(255)
   address      String   @db.VarChar(255)
   latitude     Decimal @db.Decimal(9, 6)
   longitude    Decimal @db.Decimal(9, 6)
+  google_place_id String? @unique @db.VarChar(255)
+  google_map_uri String? @db.Text
+  website_uri String? @db.Text
+  regular_opening_hours String? @db.Text
+  formatted_address String? @db.VarChar(255)
+  photos String? @db.Text
```

Comment:
Why photos? We already have Photos tables to store relationship between a places and their images